staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -103,7 +103,7 @@ func DeleteResource(r rest.GracefulDeleter, allowsOptions bool, scope *RequestSc
 				trace.Step("Decoded delete options")
 
 				objGV := gvk.GroupVersion()
-				audit.LogRequestObject(req.Context(), obj, objGV, scope.Resource, scope.Subresource, scope.Serializer)
+				audit.LogRequestObject(req.Context(), obj, objGV, scope.Resource, scope.Subresource, metainternalversionscheme.Codecs)
```

Comment:
Can you please add some unit test for these?

Reply:
Thanks for your suggestion, I have added Unit test to this modification. PTAL, thx~

Reply:
/ping @wojtek-t

Reply:
The test you added doesn't verify the fix you're making in this PR.

I would like to see some test that fails before this change and is passing with this change...